Share The Load/Screens/ViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ViewController.swift
+//  Share The Load
+//
+//  Created by Christopher Hefferman on 10/01/2021.
+//
+
+import UIKit
+import Firebase
+import RealmSwift
+
+class ViewController: UIViewController {
+    
+    private let getRealmQuestionsButton = UIButton()
+    private let removeFromRealmButton = UIButton()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        fetchQuiz()
+        createGetRealmQuestionsButton()
+        createRemoveButton()
+    }
+    
+    
+    private func createGetRealmQuestionsButton() {
+        view.addSubview(getRealmQuestionsButton)
+        getRealmQuestionsButton.translatesAutoresizingMaskIntoConstraints = false
+        getRealmQuestionsButton.setTitle("Get Questions From Realm", for: .normal)
+        getRealmQuestionsButton.frame = view.frame
```

Comment:
Shouldn't need to do this if you're using constraints below

Reply:
Got it, removed.